src/js/components/CheckBox/doc.js

```diff
@@ -42,6 +43,14 @@ export const doc = CheckBox => {
       will be passed a React event object. The current state can be accessed
       via event.target.checked. Same as React <input onChange={} />.`,
     ),
+    pad: padPropType
+      .description(
+        `The amount of padding on the container 
+    surrounding the checkbox and its label. An object can be specified to 
+    distinguish horizontal padding, vertical padding, and padding on a 
+    particular side of the container.`,
+      )
+      .defaultValue('none'),
```

Comment:
I don't think we are setting the default to 'none' anywhere.

Reply:
You're right, I can update to undefined.

Reply:
Done.